src/utils/useTraceUpdate.jsx

```diff
@@ -0,0 +1,17 @@
+import { useRef, useEffect } from "react";
+
+export default function useTraceUpdate(props) {
+  const prev = useRef(props);
+  useEffect(() => {
+    const changedProps = Object.entries(props).reduce((ps, [k, v]) => {
+      if (prev.current[k] !== v) {
+        ps[k] = [prev.current[k], v];
+      }
+      return ps;
+    }, {});
+    if (Object.keys(changedProps).length > 0) {
+      console.log("Changed props:", changedProps);
+    }
+    prev.current = props;
+  });
+}
```

Comment:
Eg veit at det finnes en `useTraceUpdate()`-hook i React. Koffer treng vi den her? Eventuelt ikke ha samme navn

Reply:
Vent litt med å svar på det her!

Reply:
Sikker? e finn ikkje noken hook i react med de navnet

Reply:
Den e jo stjelt fra https://github.com/damiangreen/use-trace-update#readme

Burde isåfall krediteres!

Reply:
Eventuelt bære bruk npm-packagen hannes.

Reply:
Skal fikse. Men faktisk e det Miklos Jakab sin kode, og Damian Green stjal den å publiserte på npm 
https://stackoverflow.com/questions/41004631/trace-why-a-react-component-is-re-rendering

Reply:
Ka e tilstand på følelsan tell Milkos Ja Kebab?